src/test/resources/TestCase.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestCase">
+
+    <test name="Test Case" preserve-order="false">
+        <classes>
+            <class name="Homework18"/>
```

Comment:
Hello,
Your homework 18 has been reviewed, In the assignment required add Homework18 class to the xml file and specify the @Test method to be ran in the xml file. But in the your code, I can't see define test method for xml file. Please add it.
//HINT: Use the <methods> & <include> tags

![image](https://github.com/Tinabihdash/A51-QA-Automation-Framework/assets/144403954/5df3d011-df0f-420c-a3cc-b126747a8326)

Please add method and include tag for xml file. 
Best wishes from the HW team (Khanh, khanh@testpro.io)